### ISSUE TITLE
Don't display 'Confidential Counselors' email adresses with mailto: link

### ISSUE THREAD

**Issue by Riscky** [svsticky/static-sticky#319]
The email addresses from the website are being scraped. This is not a huge problem for the board, who use a good spam filter, but it is for the confidential counselors, who are just members that use their personal email.
Instead of displaying the email addresses with the mailto: property, we should display them as plain text, and split up the addresses in separate parts so crawling becomes harder.

**Comment by SpookyBoy99** [comment]
Is this still an issue as the confidential counselors now get their own custom email address via Google Groups and therefor use the same spam filter as the board does. Furthermore, their email will be deleted when they stop as counselor

**Comment by SamStaijen** [comment]
@SpookyBoy99 is that a question, or a statement? I beleave the issue is resolved if they get their own sticky mailadresses (which is the case as can be seen on https://svsticky.nl/en-US/vereniging/vertrouwenspersonen). I beleave this issue can be closed, do you agree?

**Comment by SpookyBoy99** [comment]
I agree, I haven't received any spam email in my time as confidential contact person so I think Google's spam filter is doing its magic and this can be considered a non-issue
